README.md

```diff
@@ -39,41 +47,41 @@ As per [JSON-RPC-2.0](https://www.jsonrpc.org/specification) specification reque
 - Responses **MUST** have an `id` member valued exactly as the `id` member of the request this response is for
 - Notifications **MUST NOT** have an `id` member
 
-### JSON-RPC-2.0 Defiances
+### JSON-RPC-2.0 Definitions
```

Comment:
The term "defiances" is intentionally meant as the opposite of "compliances".
This section describes where this implementation **differs** from official JSON-RPC 2.0